cdip_connector/core/connector_base.py

```diff
@@ -90,11 +90,11 @@ async def extract_load(
                     }
                 )
 
-                await self.load(session, extracted)
-                await self.update_state(session, integration)
-
+                await self.load(extracted)
                 total += len(extracted)
 
+        await self.update_state(integration)
```

Comment:
It did not make sense to update the integration state at each iteration, so it's moved outside this loop.